packages/api/src/platforms/vtex/resolvers/objectOrString.ts

```diff
@@ -0,0 +1,46 @@
+import type { GraphQLScalarSerializer } from 'graphql'
+import { GraphQLScalarType } from 'graphql'
+import { Kind } from 'graphql/language'
+
+export const ObjectOrString = new GraphQLScalarType({
+  name: 'ObjectOrString',
+  description:
+    'A string or the string representation of an object (a stringified object).',
+  parseValue: toObjectOrString,
+  serialize: stringify,
```

Comment:
I didn't get it, does the ouptut will be always a string, either a string or a stringified object?

Reply:
It is always a string, but it can be a regular string eg. `"hello world"` or and stringified object `"{"key": "value"}"`